packages/account/src/Configs/currency-selector-config.tsx

```diff
@@ -1,6 +1,6 @@
+import React from 'react';
```

Comment:
This is not a JSX file, we don't need to make it tsx

Reply:
yes, but we use here React types